source/models/persisting-records.md

```diff
@@ -24,76 +24,127 @@ store.find('post', 1).then(function (post) {
+record. For a `post` record the `RESTAdapter` would generate the a url
+`/posts/1`. Ther RESTAdapter will then use the HTTP `PUT` verb to send
+the serialized record to the `/posts/1` url.
+
+The default `RESTSerializer` will serialize the post record using the
+same format that it expects to receive when loading the record. The
+only difference is the `RESTSerializer` will not include any
+sideloaded records in the update payload.
+
+```json
+{
+  "post": [{
+    "id": 1,
+    "title": "A new post",
+    "tag": "rails",
+    "comments": [1, 2]
+  }]
+}
+```
 
-```javascript
-var post = store.createRecord('post', {
-  title: 'Rails is Omakase',
-  body: 'Lorem ipsum'
-});
+#### Server Response to a updateRecord request
+
+The server is expected to respond with the serialized record now that
+it has been updated. You may also include sideloaded records in the
+response to an update request.
+
+
+```json
+{
+  "post": [{
+    "id": 1,
+    "title": "A new post",
+    "tag": "rails",
+    "comments": [1, 2]
+  }],
+  "comments": [{
+    "id": 1,
+    "body": "FIRST",
+    "post": 1,
+  }, {
+    "id": 2,
+    "body": "Rails is unagi",
+    "post": 1
+  }]
+}
+```
 
-var self = this;
 
-var onSuccess = function(post) {
-  self.transitionToRoute('posts.show', post);
-};
+#### Customizing the Adapter updateRecord request
 
-var onFail = function(post) {
-  // deal with the failure here
-};
+Ember Data's store calls the Adapter's `updateRecord` method to update
+a record that already exists on the backend server. You can override
+this method to change how Ember Data talks with your backend when
+updating a record.
 
-post.save().then(onSuccess, onFail);
+```app/adapters/post.js
+export default DS.Adapter.extend({
+  updateRecord: function(store, type, snapshot) {
+    var data = {};
+    var serializer = store.serializerFor(type.typeKey);
 
-// => POST to '/posts'
-// => transitioning to posts.show route
+    serializer.serializeIntoHash(data, type, snapshot);
+
+    var id = get(record, 'id');
+
+    // fictional backend that requires a PATCH instead of a PUT
+    return this.ajax('posts/' + id, "PATCH", { data: data });
+  }
+});
 ```
 
-You can read more about promises [here](https://github.com/tildeio/rsvp.js), but here is another
-example showing how to retry persisting:
+#### Customizing extracting the Find response with the Serializer
 
-```javascript
-function retry(callback, nTimes) {
-  // if the promise fails
-  return callback().catch(function(reason) {
-    // if we haven't hit the retry limit
-    if (nTimes-- > 0) {
-      // retry again with the result of calling the retry callback
-      // and the new retry limit
-      return retry(callback, nTimes);
-    }
- 
-    // otherwise, if we hit the retry limit, rethrow the error
-    throw reason;
-  });
-}
- 
-// try to save the post up to 5 times
-retry(function() {
-  return post.save();
-}, 5);
+Ember Data uses the RESTSerializer's `extractUpdateRecord` method to
```

Comment:
I'm not sure. I tired to focus on the `RESTSerializer` because its the default. The `JSONSerializer` isn't mentioned until the customizing serializers guide because I think the inheritance tree of the serializers is more of an advanced topic / implementation detail.